x-pack/plugins/stack_connectors/server/connector_types/bedrock/bedrock.ts

```diff
@@ -328,7 +328,6 @@ const formatBedrockBody = ({
 
 /**
  * Ensures that the messages are in the correct format for the Bedrock API
- * Bedrock only accepts assistant and user roles.
```

Comment:
this should never have been committed, Bedrock accepts system as well. Changed the code to handle system already, but this comment got left behind.